index.php

```diff
@@ -4,21 +4,10 @@
  * Plugin Name: Gutenberg Block Styles
  * Plugin URI: https://github.com/Automattic/gutenberg-block-styles/
  * Description: A simple plugin to demonstrate how to add block styles to Gutenberg.
- * Version: 1.0
- * Author: Kjell Reigstad
+ * Version: 1.1
+ * Author: Automattic
  */
 
-/**
- * Enqueue Block Styles Javascript
- */
-function block_styles_enqueue_javascript() {
-	wp_enqueue_script( 'block-styles-script',
-		plugins_url( 'block.js', __FILE__ ),
-		array( 'wp-blocks')
-	);
-}
-add_action( 'enqueue_block_editor_assets', 'block_styles_enqueue_javascript' );
-
 /**
```

Comment:
`register_block_style()` will take care of enqueueing a registered stylesheet automatically if a handle is specified. We should probably also run it all on a hook, but I'm not sure which one is preferred for this.

```suggestion
/**
 * Register Custom Block Styles
 */
if ( function_exists( 'register_block_style' ) ) {
	function block_styles_register_block_styles() {
		/**
		 * Register stylesheet
		 */
		wp_register_style(
			'block-styles-stylesheet',
			plugins_url( 'style.css', __FILE__ ),
			array(),
			'1.1'
		);

		/**
		 * Register block style
		 */
		register_block_style(
			'core/paragraph',
			array(
				'name'         => 'blue-paragraph',
				'label'        => 'Blue Paragraph',
				'style_handle' => 'block-styles-stylesheet',
			)
		);
	}

	add_action( 'init', 'block_styles_register_block_styles' );
}
```

Reply:
> We should probably also run it all on a hook, but I'm not sure which one is preferred for this.

The [handbook](https://developer.wordpress.org/block-editor/developers/filters/block-filters/#register_block_style) doesn't provide guidance, but `init` makes sense to me.